File: c-projects/calculator.c
#include <stdio.h>
int main () {
    float n1,n2;
    char opt;
printf("enter value for first number \n");
scanf("%f",&n1);

printf("enter value for second number \n");
scanf("%f",&n2);

printf("enter operator (i.e. enter + , - ,* or /) :\n ");
scanf(" %c",&opt);

switch(opt) {
    
    case '+':
    printf("sum of given two digit is %f ",n1+n2);
    break;    

    case '-':
    printf("sum of given two digit is %f ",n1-n2);
    break;

    case '*':
    printf("sum of given two digit is %f ",n1*n2);
    break;

    case '/':
    if(n2 ==0)
    {
        printf("dividion by 0 is not possible");
    }else
    {printf("sum of given two digit is %f ",n1/n2);
    }
    break;

    default:
    printf("enter valid input");
    break;
}



}

File: py-practice-codes/calculator.py
num1 =int(input("enter first number : "))
print("first number is : ",num1)

num2 = int(input("enter second number : "))
print("second number is : ",num2)

opt = input("enter operator i.e +,-,*,/ : ")
print("your operator is : ",opt)

if opt == '*':
    print("sum is " ,num1 * num2)

elif opt == '+':
    print("sum is " ,num1 + num2)

elif opt == '-':
    print("sum is " ,num1 - num2)

elif opt == '/':
    print("sum is " ,num1 / num2)

else :
    print("encountered an unexpected error  ")
    print("kindly enter a valid operator")